developer/installation/installation-windows.md

```diff
@@ -0,0 +1,124 @@
+# Installation for Windows
+
+## Install prerequisites:
+
+**Note: These commands all assume you are running them from an administrator shell**
+
+### Install node
+Download and run the installer from the [NodeJs site](https://nodejs.org)
+
+### Install Chocolately
+
+Visit the [Chocolatey site](https://chocolatey.org/install) and follow the instructions
+
+
+### Install Git
+
+```sh
+choco install git
+````
+
+### Install Meteor
+```sh
+choco install meteor
+```
+
+### Install ImageMagick (Optional but recommended)
```

Comment:
Move Imagemagick to the bottom, after Windows Build Tools 2015, since it is not required.